src/main/java/org/folio/circulation/domain/Policies.java

```diff
@@ -9,7 +9,7 @@
   private final OverdueFinePolicy overdueFinePolicy;
   private final LostItemPolicy lostItemPolicy;
 
-  Policies(LoanPolicy loanPolicy, OverdueFinePolicy overdueFinePolicy,
+  public Policies(LoanPolicy loanPolicy, OverdueFinePolicy overdueFinePolicy,
```

Comment:
Why does this need to be public?

Reply:
to use this constructor in LoanBuilder class